### PR TITLE
Add postcard and comment in order placement

### PR DESCRIPTION
**Pull request by Svitlana-G**
image

### CODE REVIEW

File: frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+const PostcardAndComment = ({ postcardData, setDataOnSubmit }) => {
+  const [showPostcardText, setShowPostcardText] = useState(
+    postcardData?.postcard ? postcardData.postcard === 'Add postcard' : true,
+  );
+  const [showComment, setShowComment] = useState(postcardData?.comment !== '');
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      postcard: postcardData?.postcard ? postcardData.postcard : 'Add postcard',
```
Comment (halynarydasova) [inline frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx:14]: Here you can simply use postcardData?.postcard || 'Add postcard'

File: frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            setShowPostcardText(false);
+          }}
+        />
+        Без листівки
+      </label>
+      <div
+        className={`${styles.comment} ${showComment ? styles.commentOpen : ''}`}
+      >
+        <label htmlFor="comment" onClick={() => setShowComment(true)}>
+          Коментар до замовлення
```
Comment (halynarydasova) [inline frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx:69]: Are you sure it should be label? The comment field isn't hidden by default. And it doesn't hide comment field on click

File: frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx (hunk 1 above)
Comment (Svitlana-G) [reply frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx]: Done.

File: frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx (hunk 2 above)
Comment (Svitlana-G) [reply frontend/src/components/Order/components/PostcardAndComment/PostcardAndComment.jsx:74]: This text is associated to the textarea below. So I used the label.
Comment field is already corrected.

**Pull request merged**